docker/src/main/java/brooklyn/entity/container/docker/DockerContainerImpl.java

```diff
@@ -450,7 +450,7 @@ public DockerContainerLocation createLocation(Map flags) {
 
         try {
             // Create a new container using jclouds Docker driver
-            JcloudsSshMachineLocation container = host.getJcloudsLocation().obtain(dockerFlags);
+            JcloudsMachineLocation container = (JcloudsMachineLocation) host.getJcloudsLocation().obtain(dockerFlags);
```

Comment:
Keep cast but change type to `JcloudsSshMachineLocation` as we know this is what will be returned from the jclouds Docker driver at present.